doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib

```diff
@@ -354,6 +355,34 @@ The list of builtin attributes is as follows:
 Like regular attributes, they can be  used in `q` filters and in `orderBy`.
 However, they cannot be used in resource URLs.
 
+## Special Metadata Types
+
+Generally speaking, user-defined metadata types are informative; they are processed by the NGSIv2
+server in an opaque way. Nonetheless, the types described below are used to convey a special
+meaning:
+
+* `DateTime`:  identifies dates, in ISO8601 format. This Metadata can be used with the query
+  operators greater-than, less-than, greater-or-equal, less-or-equal and range. For instance
+  (only the referred attribute metadata is shown):
+
+```
+"metadata": {
+      "dateCreated": {
+        "value": "2019-09-23T03:12:47.213Z",
+        "type": "DateTime"
+      }
+}
+```
+
+The list of Special Metadata Types is as follows:
+
+* `dateCreated` (type: `DateTime`): attribute creation date as an ISO 8601 string.
+
+* `dateModified` (type: `DateTime`): attribute modification date as an ISO 8601 string.
+
+* `dateExpires` (type: `DateTime`): attribute expiration date as an ISO 8601 string. How the server
+  controls attribute expiration is an implementation specific aspect.
```

Comment:
Looking to "Special Attribute Types" section (around L313-L314) I suggest to remove this block of line.

Reply:
I mean the block from L377-L384

Reply:
Fixed in 4f24cda